components/ILIAS/Contact/PRIVACY.md

```diff
@@ -0,0 +1,74 @@
+- After confirmation, each contact is stored, including the ID of the requesting user and the ID of the requested user as
+  well as a timestamp of the request time.
+- The storage of requests and approved contacts are separated.
+- Each contact added as an entry to a mailing list is stored by using the user_id.
+- Any user relations that originate from the contact is not related or depending on the active contact. There is no
+  traceable relation of contacts withing other components.
+
+## Data being presented
+
+- Contact data is presented within "Communication > Contacts" and within a separate section of the "Who is online?" tool.
+- The presentation includes a presentation of the user's profile (restricted depending on the user's profile publishing
+  settings) and control elements presenting the user's options for contact requests ("Confirm Contact"/"Ignore Contact").
+- Initially the options are restricted to requesting a contact.
+- The requesting user can cancel the request.
+- The requested user has the option to actively ignore or confirm the request.
+- Both contacts can unlink the contact without any further consent.
+- Each party can detect a change of the contact relation within its own contact options. The only exception to this
+  is the active ignoring of a request. The requesting user can not detect if his request was ignored.
+- A list of all contacts is shown as a selection when a user adds members to a newly created mailing list.
+- A contact relation cannot be perceived within the user interface by any user other than the contacts themselves.
+
+## Data being deleted
+
+- If a contact request is canceled, the stored entry is removed without any traceable footprint.
+- If a contact request is ignored, the flagged entry in the database will be preserved to prevent another request.
+- If a contact is unlinked by one of the contacts, the stored entry is removed without any traceable footprint.
+- Any user relations that originate from the contact are not affected by its deletion. This includes preservation in
+  mailing lists and participation within chatrooms.
+- If a user is deleted, its contacts, send and received contact requests, mailing lists and mailing list participations
+  are removed without any traceable footprint.
+
+## Data being exported
+
+- The component does not provide any export option itself.
+- A contact request can trigger a notification, which may be displayed as a "Toast" to the requested user. The 
+  notification contains the users presentation name (depending on his visibility settings) and the users intention of
+  the contact request.
+  - The notification is handled by the "Notification" component, is self persistent, and therefore not depending on the
+    origin request.
+- Otherwise, the information of contacts is not stored in any export of the related components like "Mail", "Chatroom" or "ILIASObject".
```

Comment:
We should not use `ILIASObject` here. If you mean `Services/Object` user `Object` as component name.

Reply:
The component `Object` was renamed to `ILIASObject` in ILIAS 10